datus/storage/backends/vector/factory.py

```
@@ -0,0 +1,62 @@
+    def build_lance_backend() -> VectorBackend:
+        from datus.storage.backends.vector.lance import LanceBackend
+
+        return LanceBackend(db_path)
+
+    if agent_config is not None:
+        namespace_fn = getattr(agent_config, "storage_backend_namespace", None)
+        resolve_db_config_fn = getattr(agent_config, "resolve_storage_db_config", None)
+        if callable(namespace_fn) and callable(resolve_db_config_fn):
+            try:
+                namespace_name = namespace_fn("vector")
+            except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)